iothub/service/src/ManagedIdentity.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// The managed identity used to access the storage account for IoT hub import and export jobs.
```

Comment:
@nimengan - Here is the description I added. We should keep this consistent across SDKs and in the swagger. At the moment we don't have any description for this in the swagger.

Reply:
I think we should say something like "the resource ID for the managed identity..." just to make it more clear.

Reply:
I added that below for the actual "userAssignedIdentity" property. This class might contain other things as designed by the service team and hence I kept it a little more generic. Open to other suggestions.